content/developer/reference/standard_modules/account/account_tax_report_line.rst

```diff
@@ -0,0 +1,24 @@
+.. _reference/account_tax_report_line:
+
+===============
+Tax Report line
```

Comment:
```suggestion
Tax Report Line
```

Reply:
I'm confused. Do you want sentence case or title case for the titles?

Reply:
I suggested this change because it's an Odoo model name. The [guidelines](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#titles-and-headings) recommend capitalizing *feature names* so I'm not sure about this one. I'd go for the same naming convention as in Odoo... @jcs-odoo what do you think?

Reply:
@william-andre @AntoineVDV 

Indeed, the guidelines are too blurry about the exceptions to the sentence case.
https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#titles-and-headings

I intend to change them slightly by removing the exception about feature names (but keeping the other exceptions). This should remove 99,9% of doubts and make it easier for everyone as it would be with sentence case pretty much everywhere :)

Here is an example.
![image](https://user-images.githubusercontent.com/46810843/166678097-edb58714-0136-4f73-ba94-d0450d88466f.png)

Actually, I'll make a PR for that right now :)

Reply:
Here it is ^^
https://github.com/odoo/documentation/pull/1910